src/storage/DatedIrsVamm.sol

```diff
@@ -738,7 +765,7 @@ library DatedIrsVamm {
         int256 trackerBaseTokenGrowthBetween
     )
     {
-        Tick.checkTicks(tickLower, tickUpper);
+        checkTicksLimits(tickLower, tickUpper);
 
```

Comment:
I don't see any risks. In contrary, this is likely unnecessary because `growthBetweenTicks` is used only by `getAccountFilledBalances` that loops through existing positions (where we checked ticks before creating them). For extra safety, we can leave it as it is.